did-pkh-method-draft.md

```diff
@@ -156,41 +187,56 @@ Construct the DID Document for *did* as follows:
 - Construct an empty DID Document metadata object, *docMeta*.
 - Return *resMeta*, *doc*, and *docMeta*.
 
+
 ### Update
 
-No updates possible. did:pkh DID Documents are, like [did:key] documents, intended for local-only usage.
+No updates possible. did:pkh DID Documents are, like [did:key] documents, 
+intended for local-only usage.
```

Comment:
Separate issue probably, but "intended for local-only usage" is quite weird for pkh did as well.